content/hardware/08.edu/solution-and-kits/alvik/tutorials/user-manual/user-manual.md

```diff
@@ -288,10 +1458,44 @@ You can now test using read_color_sensor.py in the examples folder. Refer to the
 
 ***Colors are tested on paper painted using acrylic marker pens, such as UNIPOSCA, or paper printed with an inkjet printer.***
 
-## Extensions
+## Expanding the Robot
+
+### Additional Connectors
+
+Alvik includes several connectors that expand its capabilities:
+
+- **Qwiic Connectors**
+- **Grove Connectors**
+- **Servomotor Connectors**
+- **Lego Technic Compatibility**
+
+![Connector positions](assets/connectorsExpantion.png)
+
+### Qwiic Connectors
+
+The Qwiic connectors on the Arduino Alvik robot are used for I2C connectivity with compatible sensors and actuators using the Qwiic standard. This standard simplifies the process of connecting multiple I2C devices by using a common connector and pinout. The Qwiic system supports daisy-chaining, allowing multiple devices to be connected in series. There are two Qwiic connectors, one on each side at the back of the robot, allowing for easy expansion and integration of additional components.
+
+### Grove Connectors
+
+The Grove connectors on the Arduino Alvik robot are used for connectivity with compatible sensors and actuators following the Grove standard. The Grove system uses standardized connectors and pinouts to simplify the connection of various I2C devices. The system is modular and user-friendly, ideal for rapid prototyping and educational purposes. There are two Grove connectors, one on each side at the back of the robot, providing additional flexibility for expanding the robot's capabilities.
+
+### Servomotor Connectors
+
+The Arduino Alvik robot features 3-pin connectors for attaching servomotors while providing PWM control external servomotors. The connectors are disposed on a 6 by 2 header in the center at the back of the robot, supporting multiple servomotor connections. This arrangement enables complex robotic movements and functions, allowing the robot to perform a variety of additional tasks.
+
+

```

Comment:
Would [this image](https://docs.arduino.cc/static/766550ac1fea3d1bbc8f39d068a64b87/798d4/datasheet_connectors.png) help here?

Reply:
Yes it is linked on 1472 but created individuals to the other sections